### DISCUSSION TITLE
Plyr-react library with Next.js 14

### DISCUSSION THREAD

**Discussion by chintan9** [vercel/next.js#58911] [Help]
### Summary

I’m currently experiencing an issue with the Plyr-react library in my Next.js project and would appreciate any assistance or guidance.

https://github.com/chintan9/plyr-react/issues/873 

I’m seeking advice on how to update my library to ensure compatibility with Next.js version 14

**Comment by timomedia** [comment]
I don't know if this is an error due to nextjs version or your source code. But this error appears in nextjs 14.0.2 and above, but 14.0.1 still works normally.
